community/index/src/main/java/org/neo4j/index/internal/gbptree/GBPTree.java

```diff
@@ -767,19 +751,10 @@ private void close( Header.Writer headerWriter ) throws IOException
                 return;
             }
 
-            try
-            {
-                writer.close();
-
-                // Perform a checkpoint before closing. If no changes has happened since last checkpoint,
-                // no new checkpoint will be created.
-                checkpoint( IOLimiter.unlimited(), headerWriter );
-            }
-            finally
-            {
-                closed = true;
-                pagedFile.close();
-            }
+            closed = true;
```

Comment:
Doesn't matter since `closed` is only read/changed inside this method, guarded by a lock.